apps/src/applab/designElements/TextAlignmentPropertyRow.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import applabMsg from '@cdo/applab/locale';
+import EnumPropertyRow from './EnumPropertyRow';
+
+export const TEXT_ALIGNMENT_LEFT = 'left';
```

Comment:
Would it make sense to define these constants in `applab/constants.js`? Something like:
```
export const TextAlignments = {
  LEFT: 'left',
  RIGHT: 'right',
  CENTER: 'center',
  JUSTIFY: 'justify',
};
```

Reply:
I believe `applab/constants.js` contains only global constants and `TEXT_ALIGNMENTS` is more specific and used only in this particular component

Reply:
I see that you import the constants in other `.jsx` files such as `designElements/button.jsx` and `designElements/dropdown.jsx`.

Reply:
@ebeastlake, what do you think about this?

Reply:
I think it's fine to leave them here if they're specific to this particular component. In my opinion, it's easier to notice duplicate constants scoped to child components and move them to a file at a higher scope after the fact; compared to putting something in a higher scope that someone might not realize is there and decides to create a local version of. We have _so_ many variants of `/constants.js` files that in our codebase at so many levels I don't feel like putting them in a constants file makes them super discoverable. @elf-code thoughts?

I'm more curious if there are other things in the `applab/constants/js` file that should be addressed in a follow-up PR, like the "New screen..." and "Import screen..." strings, see: https://github.com/code-dot-org/code-dot-org/pull/52317/files#diff-e289c2d9e761c0b085ada63546a6e73fecbd92a09d2681495951f299eb3b9c88R17

Reply:
Looking at how these are used elsewhere, I could see an argument for putting 
```
export const LEFT = 'left';
export const RIGHT = 'right';
export const CENTER = 'center';
export const JUSTIFY = 'justify';
```
somewhere higher in scope and having this component import them and use
```
const TEXT_ALIGNMENTS = [LEFT, RIGHT, CENTER, JUSTIFY];
```
I'm a little surprised those aren't defined somewhere already, let me look...

Reply:
I don't see a constant for this anywhere; this is the closest I see: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/music/redux/musicRedux.ts#L13